File: csharp/SupermarketReceipt/Discount.cs
namespace SupermarketReceipt
{

    public class Discount
    {
        public string Description { get; }
        public double DiscountAmount { get; }
        public Product Product { get; }

        public Discount(Product product, string description, double discountAmount)
        {
            this.Product = product;
            this.Description = description;
            this.DiscountAmount = discountAmount;
        }

    }
}

File: python/models/discount.py
from .product import Product

class Discount:
    def __init__(self, product: Product, description: str, discount_amount: int):
        self.product = product
        self.description = description
        self.discount_amount = discount_amount

    def __repr__(self):
        return (f"Discount(product={self.product}, "
                f"description={self.description}, discount_amount={self.discount_amount})")